src/runtime/pay-client.js

```diff
@@ -386,7 +173,9 @@ export class PayClientBindingPayjs {
     return promise;
   }
 
-  /** @override */
+  /**
+   * @param {function(!Promise<!Object>)} callback
```

Comment:
Can we make this Promise return a specific kind of object?

Reply:
Done.